R/combinations-calculate_matrix_metric.R

```diff
@@ -92,18 +92,15 @@ calculate_Bliss <- function(sa1, series_id1, sa2, series_id2, metric) {
   checkmate::assert_true(all(sa1[[series_id2]] == 0L))
```

Comment:
Param `metric` seems to be not needed.

OR You change `metric = "average"` (and adapt the rest of the code accordingly)

Reply:
Removed